rasa/core/brokers/file.py

```diff
@@ -30,11 +30,7 @@ async def from_endpoint_config(
         event_loop: Optional[AbstractEventLoop] = None,
     ) -> Optional["FileEventBroker"]:
         """Creates broker. See the parent class for more information."""
-        if broker_config is None:
-            return None
-
-        # noinspection PyArgumentList
-        return cls(**broker_config.kwargs)
+        return None if broker_config is None else cls(**broker_config.kwargs)
```

Comment:
Function `FileEventBroker.from_endpoint_config` refactored with the following changes:

- Lift code into else after jump in control flow ([reintroduce-else](https://docs.sourcery.ai/refactorings/reintroduce-else/))
- Replace if statement with if expression ([assign-if-exp](https://docs.sourcery.ai/refactorings/assign-if-exp/))